app/globals.css

```diff
@@ -69,18 +59,18 @@
   #controller {
     flex-basis: var(--controller-w-px);
   }
-
   #search {
     top: calc(100vh - (100vh - var(--screen-ratio-h-vh)));
     height: calc((100vh - var(--screen-ratio-h-vh)));
   }
-  .player-search {
-    @apply absolute w-full -top-1/5vw md:static md:bottom-auto md:w-auto;
+  #search-modal {
+    z-index: 2;
```

Comment:
It's same too!

Reply:
I remove z-index, instead I reorganize the order of JSX elements.
And delete isPlaying, is unnecessary!